Chapter 1/1 - Creating a Window/Program.cs

```diff
@@ -1,18 +1,16 @@
 ﻿namespace LearnOpenGL_TK
 {
-    public class Program
+    public static class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            using (var game = new Game(800, 600, "LearnOpenTK - Creating a Window"))
-            {
-                //To create a new window, create a class that extends GameWindow, then call Run() on it.
-                //Run takes a double, which is how many frames per second it should strive to reach.
-                //You can leave that out and it'll just update as fast as the hardware will allow it.
-                game.Run(60.0);
-            }
+            // To create a new window, create a class that extends GameWindow, then call Run() on it.
+            // Run takes a double, which is how many frames per second it should strive to reach.
+            // You can leave that out and it'll just update as fast as the hardware will allow it.
+            using (var window = new Window(800, 600, "LearnOpenTK - Creating a Window"))
+                window.Run(60.0);
```

Comment:
Revert this, it's against the style guide.

Reply:
Fixed